notebooks/cookbook/Enhanced sampling methods/Umbrella sampling MD.ipynb

```diff
@@ -0,0 +1,436 @@
+    "        self.addPerDofVariable(\"oldx\", 0)\n",
+    "        self.addGlobalVariable(\"V0\", 0)\n",
+    "        self.addPerDofVariable(\"sigma\", 0)\n",
+    "        self.addUpdateContextState();\n",
+    "        \n",
+    "        # Comment out old AMD method \n",
+    "        # self.addComputePerDof(\"v\", \"v+dt*fprime/m; fprime=f*((1-modify) + modify*(alpha/(alpha+E-energy))^2); modify=step(E-energy)\")\n",
+    "        # self.addComputePerDof(\"oldx\", \"x\")\n",
+    "        # self.addComputePerDof(\"x\", \"x+dt*v\")\n",
+    "        # self.addConstrainPositions()\n",
+    "        # self.addComputePerDof(\"v\", \"(x-oldx)/dt\")\n",
+    "\n",
+    "        # new variables for Langevin kernel\n",
+    "        self.addGlobalVariable('kT', kB * temperature)\n",
+    "        self.addComputePerDof(\"sigma\", \"sqrt(kT/m)\")\n",
+    "        self.addGlobalVariable(\"a\", np.exp(-1 * gamma)) #vscale?\n",
+    "        self.addGlobalVariable(\"b\", np.sqrt(1 - np.exp(-2 * gamma))) # noise-scale?\n",
+    "\n",
+    "        #before position restraints\n",
+    "        #self.addPerDofVariable(\"oldx\", 0)\n",
+    "\n",
+    "        # Compute steps for Langevin - attempt at VRORV splitting?\n",
+    "        self.addComputePerDof(\"v\", \"v + (dt / 2) * f/m\")\n",
+    "        # langevin like normal\n",
+    "        self.addComputePerDof(\"x\", \"x + (dt / 2)*v\")\n",
+    "        self.addComputePerDof(\"oldx\", \"x\")\n",
+    "        self.addConstrainPositions()\n",
+    "        self.addComputePerDof(\"v\", \"v + (x - oldx)/(dt / 2)\")\n",
+    "        self.addComputePerDof(\"v\", \"(a * v) + (b * sigma * gaussian)\") # taken from openmm-tutorial on custom integrators (link 1 above)\n",
+    "        self.addComputePerDof(\"x\", \"x + (dt / 2)*v\")\n",
+    "        self.addComputePerDof(\"oldx\", \"x\")\n",
+    "        self.addConstrainPositions()\n",
+    "        self.addComputePerDof(\"v\", \"v + (x - oldx) / (dt / 2)\")\n",
+    "        # Use the same line as what you calculated above\n",
+    "        self.addComputePerDof(\"v\", \"v + (dt / 2) * f/m\")\n",
+    "        # compute the V and deltaV values\n",
+    "        self.addComputeGlobal(\"V0\", \"energy\")\n",
+    "        self.addConstrainVelocities()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "integrator = LangevinEqIntegrator(dt=time_step,\n",
+    "                                  temperature=300*kelvin,\n",
+    "                                  collision_rate=1.0/picosecond)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create the simulation object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "platform = Platform.getPlatformByName('OpenCL')\n",
+    "platform.setPropertyDefaultValue('Precision', 'mixed')"
```

Comment:
Two problems:

1. This is unnecessary complexity for a cookbook entry.  It's unrelated to umbrella sampling.  The user already knows how to select a platform.
2. OpenCL is probably not available on CI.

Just omit the platform and let it select one automatically.